src/fs/delete.js

```diff
@@ -1,5 +1,24 @@
+import path from 'node:path';
+import promises from 'node:fs/promises';
+import { fileURLToPath } from 'url';
+
 const remove = async () => {
-    // Write your code here 
+    const ERROR_TEXT = 'FS operation failed';
+    const fileName = 'files/fileToRemove.txt';
+
+    const modulePath = path.dirname(fileURLToPath(import.meta.url));
+    const filePath = path.join(modulePath, fileName);
```

Comment:
Also good idea to move this function into 'utils' folder, as it's being repeated many times

Reply:
This is a great idea! Thanks. I will refactor this after the end of the cross-check so that subsequent reviewers do not deduct points for commits after the deadline.